src/cascadia/TerminalSettingsEditor/GlobalSettingsViewModel.h

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#pragma once
+
+#include "GlobalSettingsViewModel.g.h"
+#include "Utils.h"
+#include "ViewModelHelpers.h"
+#include "..\TerminalSettingsModel\MTSMSettings.h"

```

Comment:
I know... I really shouldn't just grab this from `TerminalSettingsModel`, but it lets me use the X-Macro and it's so nice! Should I move this to `inc` or something? I feel like it's fine.

Reply:
I feel like it's fine. I think the traditional OS code style would be 
```
TerminalSettingsModel/ -> contains the actual soure
  - inc/ -> has shared headers like MTSMSettings.h
  - lib/ -> builds the .lib
  - dll/ -> builds the .dll
```

but honestly, _meh_